### PR TITLE
Chinese translation added.

### PR DESCRIPTION
**Pull request by laiqun**

### CODE REVIEW

File: smartautoclicker/src/main/res/values-zh/strings.xml [hunk 1]
```diff
@@ -0,0 +1,147 @@
...
+ * along with this program; If not, see <http://www.gnu.org/licenses/>.
+-->
+<resources>
+    <string name="app_name" translatable="true">智能点击器</string>
+
+    <string name="activity_scenario_title">场景列表</string>
+    <string name="menu_search_item_title">搜索</string>
+    <string name="no_scenarios">场景列表为空，\n点击 \'+\' 按钮来创建。</string>
+    <plurals name="scenario_sub_text">
+        <item quantity="one">%d 情景</item>
```
Comment (Nain57) [inline smartautoclicker/src/main/res/values-zh/strings.xml:25]: Lint give a warning saying "one" isn't relevant in chinese.
If that's actually the case, you can remove the <item quantity="one">%d 情景</item> line

**Comment by Nain57** [comment]
Thanks again for your contribution !
Several strings are still missing, but if you just want to contribute to those, that's ok !

**Comment by laiqun** [comment]
> 

It's event_default_config.xml file, right?

File: smartautoclicker/src/main/res/values-zh/strings.xml (hunk 1 above)
Comment (laiqun) [reply smartautoclicker/src/main/res/values-zh/strings.xml:25]: Yes. Chinese don't use plurals. It use other word,such as "群,多,大量". 
I will delete this part.

**Comment by Nain57** [comment]
I think you have an old version of the application synchronized in your workspace. 
Several new menu have been added with a new set of strings. But you don't have to finish them all and already submit those

**Comment by Nain57** [comment]
Someone have provided up to date translations in another PR.
This one is not required anymore.

**Pull request closed without merging**